APISpecification.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: 3.0.3
+info:
+  title: Ensembl Resolver API
+  description: Resolver API resolves external urls to Ensembl
+  contact:
+    email: ensembl-webteam@ebi.ac.uk
+  version: 0.0.1
+servers:
+  - url: https://resolver.ensembl.org
+tags:
+  - name: resolver
+    description: Resolver API resolves external urls to Ensembl
+paths:
+  /resolve/{gene_stable_id}:
+    get:
+      summary: Resolve stable ID with optional query params
+      description: Resolves to a beta url when a stable id wih optional query params provided
+      parameters:
+        - name: gene_stable_id
+          in: path
+          description: Gene stable id used to represent a gene in a genome.
+          required: true
+          explode: true
+          schema:
+            type: string
+            example: ENSG00000139618
+        - name: type
+          in: query
+          description: Defines a type - gene/transcript/variant etc
+          required: false
+          schema:
+            type: string
+        - name: app
+          in: query
+          description: Choose between apps - Genome Browser/ Entity Viewer
+          required: false
+          schema:
+            type: string
+        - name: gca
+          in: query
+          description: GCA assembly accession id of the genome
+          required: false
+          schema:
+            type: string
+       responses:
+        '302':
+          description: Redirect to resolved URL
+          headers:
+            Location:
+              schema:
+                type: string
+          content: {}
+        '400':
+          $ref: '#/components/responses/400'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+components:
+  responses:
+    400:
+      description: Bad Request. Missing stable id in the url
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: '{"status_code": 400, "details": "Not Bad Request"}'
```

Comment:
- `Not Bad Request` is not quite correct 😄 
- You are describing content as `text/plain`, but giving an example of a json. I suspect we will want both an `application/json` for requests that expect json in return, and a `text/html` for when the url is accessed via the browser.

Reply:
does that mean this needs update too? Because I used this as a reference. :-) 
https://github.com/Ensembl/ensembl-web-metadata-api/blob/main/APISpecification.yaml#L878